torchvision/models/vision_transformer.py

```diff
@@ -26,6 +26,14 @@
 }
 
 
+class ConvStemLayer(NamedTuple):
+    out_channels: int
+    kernel_size: int
+    stride: int
+    padding: Union[str, int]
```

Comment:
Nowhere else in TorchVision we accept padding as string. If you want to estimate it automatically, we encode this by passing None.
```suggestion
    padding: int
```